### ISSUE TITLE
CHATHISTORY/PLAYBACK for SNOTICES

### ISSUE THREAD

**Issue by Mikaela** [ergochat/ergo#1706]
Currently SNOTICES are only available from either already connected clients or logs, so operators cannot really replace bouncer or remote shell with just connecting to Ergo directly.

I think we discussed this on #Ergo, but an issue wasn't opened.

**Comment by Mikaela** [comment]
Checking IRC another thing that was mentioned is massmessages as it's possible that first there is a massmessage about maintenance done during e.g. next hour and a client can connect and experience distruptions at that time without receiving the prior massmessage.

**Comment by slingamn** [comment]
Just to clarify, "SNOTICE" here refers to operator notices sent due to the `+s` usermode?

**Comment by Mikaela** [comment]
Yes, and I think seeing them is important for e.g. `chat-moderator` privset.
